thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -331,15 +331,17 @@ public Response createAnomalyFunction(@NotNull @QueryParam("dataset") String dat
       break;
     case HOURS:
     default:
-      windowDelayTime = 2;
+      windowDelayTime = 0;
       windowDelayTimeUnit = TimeUnit.HOURS;
     }
     anomalyFunctionSpec.setWindowDelayUnit(windowDelayTimeUnit);
     anomalyFunctionSpec.setWindowDelay(windowDelayTime);
 
     // setup detection frequency if it's minutely function
+    // the default frequency is determined by data granularity
+    // this frequency is a override to lower the system overload
     if (dataGranularity.getUnit().equals(TimeUnit.MINUTES)) {
-      TimeGranularity frequency = new TimeGranularity(30, TimeUnit.MINUTES);
```

Comment:
Is inGraph itself being refreshed every 30mins? Though we do real time data retrieval, but inGraph is not real time?..This is an impression I got when talking to Ads POC, correct me if I'm wrong.